arch/sparc/src/bm3803/bm3803-serial.c

```diff
@@ -238,11 +238,15 @@ static uart_dev_t g_uart1port =
 {
   .recv      =
   {
+    .head = 0,
+    .tail = 0,
```

Comment:
is sparc compiler not initing those values to zeroes by default? I think it should as this is a part if C standard

Reply:
I think you're right